packages/web-pkg/src/composables/appDefaults/useAppFolderHandling.ts

```diff
@@ -51,7 +51,9 @@ export function useAppFolderHandling({
       context = unref(context)
       const space = unref(context.space)
 
-      const pathResource = await getFileInfo(context, { davProperties: [DavProperty.Name] })
+      const pathResource = await getFileInfo(context, {
+        davProperties: [DavProperty.FileId]
+      })
```

Comment:
Don't we need DavProperty.Name for the path?!

Reply:
@dschmidt I was wrong to say that we need the `name` prop. `name` is only the display name of the file.